README.md

```diff
@@ -39,6 +39,7 @@ appium:appWorkingDir | Full path to the folder, which is going to be set as the
 appium:createSessionTimeout | Timeout in milliseconds used to retry Appium Windows Driver session startup. This capability could be used as a workaround for the long startup times of UWP applications (aka `Failed to locate opened application window with appId: TestCompany.my_app4!App, and processId: 8480`). Default value is `20000`.
 ms:waitForAppLaunch | Similar to `createSessionTimeout`, but in seconds and is applied on the server side. Enables Appium Windows Driver to wait for a defined amount of time after an app launch is initiated prior to attaching to the application session. The limit for this is 50 seconds.
 ms:experimental-webdriver | Enables experimental features and optimizations. See Appium Windows Driver release notes for more details on this capability. `false` by default.
+ms:forcequit | Defines if the WinAppDriver should be started with the `/forcequit` command line argument. Default `false`.
```

Comment:
It would also be useful to add an explanation here about the purpose of the actual command line argument

Reply:
Took the description from WAD.